cpp/lib/include/StringUtil.h

```diff
@@ -1985,6 +1985,11 @@ inline bool ConsistsOfAllASCIIDigits(const std::string &s) {
 }
 
 
+inline bool IsPrintableASCIICharacter(const char ch) {
+    return static_cast<unsigned char>(ch) >= 33 and static_cast<unsigned char>(ch) <= 126;
```

Comment:
Shouldn't a space (32) also be a printable ASCII character?

Reply:
If not, it should at least be commented

Reply:
Should be ok now...